plugins/reolink/src/reolink-api.ts

```diff
@@ -85,6 +91,21 @@ export class ReolinkCameraClient {
         this.tokenLease = Date.now() + 1000 * leaseTimeSeconds;
     }
 
+    async requestWithToken(options: HttpFetchOptions<Readable>, body?: Readable) {
+        if (!this.tokenData?.token || !this.tokenData.leaseTime || this.tokenData.leaseTime > Date.now()) {
```

Comment:
what's happening here? requestWithLogin already does token based requests

Reply:
seems you can add a parameter to requestWithLogin to force a token based request. copy pasting the entire method seems unnecessary